src/graph.py

```diff
@@ -525,6 +529,11 @@ def randomizations(self, noisy_edges_reps, random_terminals_reps):
         else: sys.exit("Randomizations was called with invalid noisy_edges_reps and random_terminals_reps.")
 
         ###########
+
+        # Empty networks breaks `output_forest_as_networkx`. 
```

Comment:
Doesn't it make sense to fix `output_forest_as_networkx` then?

Reply:
e9b73b3fd7ecc16298319fab4c115d532793276a